include/kvs/server_utils.hpp

```diff
@@ -110,9 +110,6 @@ class MemoryOrderedSetSerializer : public Serializer {
 
   string get(const Key &key, AnnaError &error) {
     auto val = kvs_->get(key, error);
-    if (val.size().reveal() == 0) {
-      error = AnnaError::KEY_DNE;
-    }
```

Comment:
@cw75 What was this code doing before? Presumably setting the error if the key doesn't exist? If we remove these lines, does the error still get set somewhere else or no?

Reply:
@xcharleslin originally this was intended to flag keys that are GCed. But then we couldn't distinguish between an empty set and a GCed set. So let's remove these check now and later when we implement GC for set, we'll set a tombstone flag for GCed keys.